telemetry/tracks/class-tracks-event.php

```diff
@@ -153,24 +152,19 @@ protected function process_properties(
 			unset( $event->_via_ip );
 		}
 
-		// Set VIP environment if it exists.
-		if ( defined( 'VIP_GO_APP_ENVIRONMENT' ) ) {
-			$app_environment = constant( 'VIP_GO_APP_ENVIRONMENT' );
-			if ( is_string( $app_environment ) && '' !== $app_environment ) {
-				$event->vipgo_env = $app_environment;
-			}
+		$base_props = get_base_properties_of_track_event();
+
+		if ( isset( $base_props['vipgo_env'] ) ) {
+			$event->vipgo_env = $base_props['vipgo_env'];
 		}
 
-		// Set VIP organization if it exists.
-		if ( defined( 'VIP_ORG_ID' ) ) {
-			$org_id = constant( 'VIP_ORG_ID' );
-			if ( is_integer( $org_id ) && $org_id > 0 ) {
-				$event->vipgo_org = $org_id;
-			}
+		if ( isset( $base_props['vipgo_org'] ) ) {
+			$event->vipgo_org = $base_props['vipgo_org'];
```

Comment:
I'm assuming we don't want to change the names of any of this stuff? Even though `vipgo` is outdated.

Reply:
There's another PR from @alecgeatches where I noted that we should clean up these prop names.

https://github.com/Automattic/vip-go-mu-plugins/pull/5922#pullrequestreview-2363403410

Reply:
Not making any change based on Mo's comment.

Reply:
Sorry, to clarify: we should rename the props. Please confirm with Alec if he's going to continue with his PR or if we should change things here.

Reply:
PR: https://github.com/Automattic/vip-go-mu-plugins/pull/5990